actionpack/lib/action_controller/api/api_rendering.rb

```diff
@@ -0,0 +1,14 @@
+module ActionController
+  module ApiRendering
```

Comment:
It seems this module was added to avoid the inclusion of `ActionView::Rendering` (this is currently being included on [rails-api](https://github.com/rails-api/rails-api/blob/master/lib/rails-api/action_controller/api.rb#L150)).

However, I tested this PR [without this module](https://gist.github.com/jmbejar/f1afbb97b807c075aa99) and it seems to work properly.

I think this still works because the render method options are processed in the `Renderers` module [here](https://github.com/spastorino/rails/blob/rails-api/actionpack/lib/action_controller/metal/renderers.rb#L43) (and api controllers still use this module).

In brief, we should be fine just including `Rendering`, no need to have a specific `ApiRendering`.